mw-webapp/src/component/editableTextarea/editableTextarea.tsx

```diff
@@ -71,18 +74,22 @@ export const EditableTextarea = (props: EditableTextareaProps) => {
     }
   };
 
+  const handleChange = (event: ChangeEvent<HTMLTextAreaElement>) => {
+    const newText = event.target.value;
+    setText(newText);
+
+    if (event.target.parentNode instanceof HTMLElement) {
+      event.target.parentNode.dataset.replicatedValue = newText;
+    }
+  };
+
   /**
    * Render Textarea
    */
   const renderTextarea = () => (
-    <Textarea
-      defaultValue={text}
-      onChange={setText}
-      placeholder={props.placeholder ?? ""}
-      rows={props.rows}
-      isAutofocus
-      onKeyPress={handleCtrlEnter}
-    />
+    <div className="grow-wrap">
+      <textarea name="text" id="text" value={text} onChange={e => handleChange(e)} ></textarea>
```

Comment:
why do we need id and name?

Reply:
I wrote this out of habit and have already deleted it